charts/substra-backend/templates/deployment-server.yaml

```diff
@@ -27,89 +27,119 @@ spec:
-            - name: BACKEND_DEFAULT_PORT
-              value: {{ .Values.backend.service.port | quote}}
-            - name: BACKEND_PEER_PORT
-              value: "internal"
-            - name: LEDGER_CONFIG_FILE
-              value: /conf/{{ .Values.organization.name }}/substra-backend/conf.json
-            - name: MEDIA_ROOT
-              value: {{ .Values.persistence.hostPath }}/medias/
-            - name: PYTHONUNBUFFERED
-              value: "1"
-          ports:
-            - name: http
-              containerPort: {{ .Values.backend.service.port }}
-              protocol: TCP
-          volumeMounts:
-            - name: data
-              mountPath: {{ .Values.persistence.hostPath }}
-            - name: config
-              mountPath: /conf/{{ .Values.organization.name }}/substra-backend
-              readOnly: true
-            - name: user-cert
-              mountPath: /var/hyperledger/msp/signcerts
-            - name: user-key
-              mountPath: /var/hyperledger/msp/keystore
-            - name: peer-tls-server
-              mountPath: /var/hyperledger/tls/server/pair
-            - name: peer-tls-client
-              mountPath: /var/hyperledger/tls/client/pair
-            - name: orderer-tls-ca
-              mountPath: /var/hyperledger/tls/ord/cert
-            - name: ca
-              mountPath: /var/hyperledger/ca
-          livenessProbe:
-            httpGet:
-              path: /liveness
-              port: http
-              httpHeaders:
-                - name: Accept
-                  value: "text/html;version=0.0, */*;version=0.0"
-            initialDelaySeconds: 60
-            timeoutSeconds: 5
-            periodSeconds: 45
-            failureThreshold: 6
-          readinessProbe:
-            httpGet:
-              path: /readiness
-              port: http
-              httpHeaders:
-                - name: Accept
-                  value: "text/html;version=0.0, */*;version=0.0"
-            initialDelaySeconds: 5
-            timeoutSeconds: 2
-            periodSeconds: 30
-          resources:
-            {{- toYaml .Values.backend.resources | nindent 12 }}
+      - name: {{ template "substra.name" . }}
+        image: "{{ .Values.backend.image.repository }}:{{ .Values.backend.image.tag }}"
+        {{- if .Values.backend.image.pullPolicy }}
+        imagePullPolicy: "{{ .Values.backend.image.pullPolicy }}"
+        {{- end }}
+        command: ["/bin/bash"]
+        {{- if eq .Values.backend.settings "prod" }}
+        args: ["-c", "python manage.py collectstatic --noinput && uwsgi --http :{{ .Values.backend.service.port }} --module backend.wsgi --static-map /static=/usr/src/app/backend/statics --master --processes 4 --threads 2 --need-app --env DJANGO_SETTINGS_MODULE=backend.settings.server.prod "]
```

Comment:
you have collectstatic twice: as an init-container and inside the container command

Reply:
thanks